File: day61/minOperations.cpp
#include<bits/stdc++.h>
using namespace std;

int minOperations(vector<int>& nums) {
        int result = 0 ;
        for(int i = 0 ; i < nums.size() ; i++) {
            if(i+1 < nums.size()){
                if(nums[i]>=nums[i+1]){
                    int n = ((nums[i]-nums[i+1]) + 1) ;
                    result+= n;
                    nums[i+1] += n ;
                }
            }
        }
        return result ;
    }

int main() {
    vector<int> nums = {1, 5, 2, 4, 1};
    cout << minOperations(nums) << endl; 
    return 0;
}

File: day87/MinOperations.py
def minOperations(nums, k) -> int:
        n = sum(nums) 
        result = 0 
        if(n% k == 0) :
            return 0 
        while(n% k != 0 and n>0) :
            n-=1 
            result+=1
        return result 
    
print(minOperations([3,9,7] , 5))